webpack/webpack.config.babel.js

```diff
@@ -0,0 +1,67 @@
+import autoprefixer from 'autoprefixer';
+import colorLighten from 'less-color-lighten';
+import path from 'path';
+
+function absPath() {
+  let args = [].slice.apply(arguments);
+  args.unshift(__dirname, '..');
+
+  return path.resolve.apply(path.resolve, args);
+}
+
+// Can override this with npm config set dcos-ui:external_plugins ../some/relative/path/to/repo
+let pluginsDir = absPath(process.env.npm_package_config_external_plugins || 'plugins');
```

Comment:
I think it's ok for the build to fail. The goal is to build the application the way you have your config set rather than building the application in a different way that what you'd want.